app/src/main/java/org/wikipedia/feed/FeedFragment.java

```diff
@@ -220,9 +235,21 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 && resultCode == SettingsActivity.ACTIVITY_RESULT_LANGUAGE_CHANGED)
                 || requestCode == ACTIVITY_REQUEST_ADD_A_LANGUAGE) {
             refresh();
+        } else if (requestCode == ACTIVITY_REQUEST_DESCRIPTION_EDIT && resultCode == RESULT_OK) {
+            if (suggestedEditsCardView != null) {
+                suggestedEditsCardView.showAddedDescriptionView(data.getStringExtra(EXTRA_SOURCE_ADDED_DESCRIPTION));
+            }
+            FeedbackUtil.showMessage(this, R.string.description_edit_success_saved_snackbar);
+        } else if (requestCode == ACTIVITY_REQUEST_SUGGESTED_EDITS_ONBOARDING && resultCode == RESULT_OK) {
+            startDescriptionEditScreen();
         }
     }
 
+    private void startDescriptionEditScreen() {
+        startActivityForResult(DescriptionEditActivity.newIntent(requireContext(), descriptionEditPageTitle, null, true, sourceDescription, sourceLangCode, suggestedEditsCardView.isTranslation() ? FEED_CARD_SUGGESTED_EDITS_TRANSLATE_DESC : FEED_CARD_SUGGESTED_EDITS_ADD_DESC),
```

Comment:
The `descriptionEditPageTitle` should set the **added** `description` to it after the user added a description and back to the feed.

Reply:
This was working before and that's the reason for:
```
if (suggestedEditsCardView != null) {              suggestedEditsCardView.showAddedDescriptionView(data.getStringExtra(EXTRA_SOURCE_ADDED_DESCRIPTION));
 }
```

Something has changed to reload feed after the edit... will look into this..

Reply:
Ah! when the 'Don't keep activities' is turned on, it refreshed the feed and in this case a whole new suggestion card is loaded. so we do not need to add the 'Added by you / Translated by you' label.. otherwise this should work..

Reply:
I meant to add a value into the `descriptionEditPageTitle`.

something like `descriptionEditPageTitle.setDescription(xxx)`.

Reply:
Oh ok done.